mantidproject/mantid
# Docs Updates after release 6.5

**Description of work.**
Add superplot to 06_formatting_plots page
Mention changing layout on gettingstarted/mainwindow basic course 
Add ADS import to 02_pim_sol solution code
Update 03_fit_model_choices to give better fit for EMU data

**To test:**

Build the docs-html target and check that the new links, images and wording are all correct!

Fixes #34377
Fixes #31682

*This does not require release notes* because **minor documentation update**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
